features/dd-sdk-android-trace/transitiveDependencies

```diff
@@ -1,13 +1,28 @@
 Dependencies List
 
 androidx.annotation:annotation:1.3.0                            :   30 Kb
+com.github.spotbugs:spotbugs-annotations:4.2.0                  :   14 Kb
+com.github.spotbugs:spotbugs:4.2.0                              :    3 Mb
+com.google.code.findbugs:jsr305:3.0.2                           :   19 Kb
 com.google.code.gson:gson:2.10.1                                :  276 Kb
 io.opentracing:opentracing-api:0.32.0                           :   18 Kb
 io.opentracing:opentracing-noop:0.32.0                          :   10 Kb
 io.opentracing:opentracing-util:0.32.0                          :   10 Kb
+net.jcip:jcip-annotations:1.0                                   :    2 Kb
+org.apache.bcel:bcel:6.5.0                                      :  678 Kb
+org.apache.commons:commons-lang3:3.11                           :  564 Kb
+org.apache.commons:commons-text:1.9                             :  211 Kb
```

Comment:
wondering if we can get rid of this, seems too heavy

Reply:
yeah I have a task for that (the one that is required to fix that lint rule). I suspect is either this dep or moshi so I plan to remove both.